clang/lib/CIR/Dialect/Transforms/LoweringPrepare.cpp

```diff
@@ -23,8 +25,9 @@
 #include "llvm/Support/ErrorHandling.h"
 #include "llvm/Support/Path.h"
 
+using cir::CIRBaseBuilderTy;
 using namespace mlir;
-using namespace cir;
+using namespace mlir::cir;
```

Comment:
Sounds beneficial, thanks. Changes like this cause a bunch of churn in the diff and make it harder to review, probably best to put them into separated PRs next time around.

Reply:
I needed to do it because of `CIRBaseBuilder` usage. Once I included a header with `cir` namespace, `using namespace cir` became unresolved, what exact `cir` is in question: either `cir` itself or `mlir::cir`